phpunit.xml.dist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="tests/bootstrap.php"
```

Comment:
Alignment looks foobar'd. Also, do you really need a `bootstrap.php`? That seems to indicate some issues with the setup (side effects)